src/pages/token-metadata/token-2022.md

```diff
@@ -301,7 +301,7 @@ These are extensions that can be enabled on mint accounts of SPL Token-2022.
 
   {% table %}
 
-  - Asset {% width="20%" %}
+  - NFT {% width="20%" %}
   - Fungible {% width="40%" %} {% align="center" %}
   - Non-Fungible {% width="40%" %} {% align="center" %}
```

Comment:
I think these tables don't make sense saying NFT because the Asset is meant to be the broader category of Fungible and Non-Fungible.  So NFT does not apply to both.